trie/verkle.go

```diff
@@ -163,14 +163,14 @@ func (t *VerkleTrie) UpdateAccount(addr common.Address, acc *types.StateAccount,
 		stem      = t.pointCache.GetTreeKeyBasicDataCached(addr[:])
 	)
 
-	binary.LittleEndian.PutUint64(basicData[utils.BasicDataNonceOffset:], acc.Nonce)
+	binary.BigEndian.PutUint64(basicData[utils.BasicDataNonceOffset:], acc.Nonce)
```

Comment:
Switch endianess as described in the lastest 4762 spec PR.

Reply:
Other changes might be needed too. I'd recommend merge this PR anyway and continue the work on the main nyota branch.